These tests are just checking the syntax of the version name, which
Alpine has its own format for.
--- thelounge-4.2.0/test/src/helperTest.js
+++ thelounge-4.2.0/test/src/helperTest.js
@@ -42,11 +42,11 @@
 	describe("#getVersion()", function () {
 		const version = Helper.getVersion();

-		it("should mention it is served from source code", function () {
+		xit("should mention it is served from source code", function () {
 			expect(version).to.include("source");
 		});

-		it("should include a short Git SHA", function () {
+		xit("should include a short Git SHA", function () {
 			expect(version).to.match(/\([0-9a-f]{7,11} /);
 		});

